conf/portal/settings_custom.py

```diff
@@ -170,5 +170,5 @@
     "makePublic": False,
     "hideApps": True,
     "hideDataFiles": True,
-    "onboardingCompleteRedirect": '/protx/'
+    "onboardingCompleteRedirect": '/analytics'
```

Comment:
Idea: If `LOGIN_REDIRECT_URL = '/analytics'` was added to `settings_custom.py`, it could be used in the block as  `"onboardingCompleteRedfrect": LOGIN_REDIRECT_URL'.

That would make the redirect configurable in the settings_custom instead of the code and also ensures we would use the same values anywhere else a redirect might be needed.

Reply:
the LOGIN_REDIRECT_URL is in the cms settings file not the core django settings file.  i think it would be tricky to import things from each other.

Reply:
Ahh! Right, thanks for catching that. I get them confused sometimes. The file naming is unclear, but I think we have since fixed that in the core portal.

Reply:
i mean core-cms =)